pgmpy/models/LinearGaussianBayesianNetwork.py

```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 from sklearn.linear_model import LinearRegression
 
+from pgmpy.models import DAG
```

Comment:
@kollisaisiddartha I think the issue is here. The import should be `from pgmpy.base import DAG`. Trying to import `pgmpy.models` is running into a circular import.